### DISCUSSION TITLE
[Script request] Discourse Forum

### DISCUSSION THREAD

**Discussion by Mati-l33t** [community-scripts/ProxmoxVE#154] [Request script]
### Application Name

Discource

### Website

https://github.com/discourse/discourse.    

### Description

Im running multiple Discource forums and also Flarum forums. It would be nice to have a script for this to speed things up. Maybe the easiest way would be to add to Dockge script and option to install also Discource or Flarum? Just like there is a option to add Immich and/or Home Assistant in the Dockge script?
### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by mzb2xeo** [comment]
Installing discourse requires a domain being setup and active. They have a good install guide available : 

https://github.com/discourse/discourse/blob/main/docs/INSTALL-cloud.md

**Comment by MickLesk** [comment]
I have “failed” with the script before, so it works, BUT: It requires a complete mail configuration beforehand, which is damn difficult to realize as a helper script, almost impossible. So much has to be done manually. NodeBB, for example, works perfectly for me locally.

Or do you have an Idea @michelroegl-brunner

**Reply by michelroegl-brunner** [reply]
Puh, this looks like two much effort to be honest. I see many issues incoming with this script. 
The domain setup at one hand and on the other end there is only the dockerfile, wich has to be bisected and native rebuild. I guess this script would be a mess regarding needed user input. Wich would break convention of our scripts. Or we support a config file for this case?
I think it will create more problems than it will solve.
